src/Illuminate/Cache/RedisStore.php

```diff
@@ -253,7 +253,7 @@ public function flush()
     /**
      * Remove all expired tag set entries.
      *
-     * @return bool
+     * @return void
```

Comment:
I don't think this is the right fix. The result of each flushState should be `&&`ed together, and the result returned.